plugins/obs-vst/obs-vst.cpp

```diff
@@ -85,10 +85,8 @@ std::string getFileMD5(const char *file)
 		if (hash.addData(&f))
 			return std::string(hash.result().toHex());
 	}
-
 	return std::string();
 }
-
```

Comment:
of course, will make sure to remove these

Reply:
thanks @RytoEX !
I see your point, I can change the name of the int hash, to uniqueID, since this is what the value stores.

setUniueID() is always called from the plugin. Not sure what you meant by "their assigned ID", the host doesn't determine the plugin's uniqueID, but the plugin sets one for itself. The old (and not functioning) Steinberg attempt to register IDs uniquely is not that worrying, since we are still storing the full dll path. So the only conflict could be for plugins which used the exact same dll name, and as the os doesn't allow two files with the same name, this guarantees uniqueness. An alternative is not to load per dll name, but by combination of uniqueID and vendor name.

When I write "should", I mean that I tested it locally and it solves it. There might be some scenarios and cases which I didn't think of and didn't cover when testing. If the dll name changes, the plugin won't load (see the alternative proposal in last paragraph). But it is not often that an update changes the plugin name. So the current fix should solve most practical cases.

Of course, I will try to follow more carefully regarding messages and titles. will squash everything properly after I make the adjustments required from your input.